cst438-project-airline/src/main/java/cst438/domain/Flight.java

```diff
@@ -27,18 +30,22 @@
 
   public Flight() {}
 
-  public Flight(int flightId, String airlineName, Date departureDateTime, int numberOfStops,
-      String originCity, String destinationCity, int price) {
-    super();
```

Comment:
Is there a reason that the line "super();" was removed? Or was that an accident?

Reply:
I added `super()` back in. It was accidentally removed.